hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterClockImpl.java

```diff
@@ -44,6 +44,11 @@ public long getClusterTime() {
         return Clock.currentTimeMillis() + clusterTimeDiff;
     }
 
+    @Override
+    public long getApproximateClusterTime() {
+        return Clock.approximateTimeMillis() + clusterStartTime;
```

Comment:
`Clock.approximateTimeMillis() + clusterTimeDiff` ?

Reply:
oops

Reply:
It is fixed.